terraform/environments/digital-prison-reporting/modules/dms/config/nomis-replication-settings-experimental.json

```diff
@@ -0,0 +1,195 @@
+    "CtrlStreamBufferSizeInMB": 5,
+    "StreamBufferSizeInMB": 8
+  },
+  "ErrorBehavior": {
+    "FailOnNoTablesCaptured": false,
+    "ApplyErrorUpdatePolicy": "LOG_ERROR",
+    "FailOnTransactionConsistencyBreached": false,
+    "RecoverableErrorThrottlingMax": 1800,
+    "DataErrorEscalationPolicy": "SUSPEND_TABLE",
+    "ApplyErrorEscalationCount": 0,
+    "RecoverableErrorStopRetryAfterThrottlingMax": true,
+    "RecoverableErrorThrottling": true,
+    "ApplyErrorFailOnTruncationDdl": false,
+    "DataTruncationErrorPolicy": "LOG_ERROR",
+    "ApplyErrorInsertPolicy": "LOG_ERROR",
+    "EventErrorPolicy": "IGNORE",
+    "ApplyErrorEscalationPolicy": "LOG_ERROR",
+    "RecoverableErrorCount": -1,
+    "DataErrorEscalationCount": 0,
+    "TableErrorEscalationPolicy": "STOP_TASK",
+    "RecoverableErrorInterval": 5,
+    "ApplyErrorDeletePolicy": "IGNORE_RECORD",
+    "TableErrorEscalationCount": 0,
+    "FullLoadIgnoreConflicts": true,
+    "DataErrorPolicy": "LOG_ERROR",
+    "TableErrorPolicy": "SUSPEND_TABLE"
+  },
+  "TTSettings": {
+    "TTS3Settings": null,
+    "TTRecordSettings": null,
+    "EnableTT": false
+  },
+  "FullLoadSettings": {
+    "CommitRate": 10000,
+    "StopTaskCachedChangesApplied": false,
+    "StopTaskCachedChangesNotApplied": false,
+    "MaxFullLoadSubTasks": 8,
+    "TransactionConsistencyTimeout": 600,
+    "CreatePkAfterFullLoad": false,
+    "TargetTablePrepMode": "DO_NOTHING"
+  },
+  "TargetMetadata": {
+    "ParallelApplyBufferSize": 0,
+    "ParallelApplyQueuesPerThread": 0,
+    "ParallelApplyThreads": 0,
+    "TargetSchema": "",
+    "InlineLobMaxSize": 0,
+    "ParallelLoadQueuesPerThread": 0,
+    "SupportLobs": true,
+    "LobChunkSize": 64,
+    "TaskRecoveryTableEnabled": false,
+    "ParallelLoadThreads": 8,
+    "LobMaxSize": 32,
+    "BatchApplyEnabled": false,
+    "FullLobMode": false,
+    "LimitedSizeLobMode": true,
+    "LoadMaxFileSize": 0,
+    "ParallelLoadBufferSize": 1000
+  },
+  "BeforeImageSettings": null,
+  "ControlTablesSettings": {
+    "historyTimeslotInMinutes": 5,
+    "HistoryTimeslotInMinutes": 5,
+    "StatusTableEnabled": false,
+    "SuspendedTablesTableEnabled": false,
+    "HistoryTableEnabled": false,
+    "ControlSchema": "",
+    "FullLoadExceptionTableEnabled": false
+  },
+  "LoopbackPreventionSettings": null,
+  "CharacterSetSettings": null,
+  "FailTaskWhenCleanTaskResourceFailed": false,
+  "ChangeProcessingTuning": {
+    "StatementCacheSize": 50,
+    "CommitTimeout": 1,
+    "BatchApplyPreserveTransaction": true,
+    "BatchApplyTimeoutMin": 1,
+    "BatchSplitSize": 0,
+    "BatchApplyTimeoutMax": 30,
+    "MinTransactionSize": 1000,
+    "MemoryKeepTime": 60,
+    "BatchApplyMemoryLimit": 500,
+    "MemoryLimitTotal": 1024
+  },
+  "ChangeProcessingDdlHandlingPolicy": {
+    "HandleSourceTableDropped": true,
+    "HandleSourceTableTruncated": true,
+    "HandleSourceTableAltered": true
+  },
+  "PostProcessingRules": null,
+  "ValidationSettings": {
```

Comment:
Validation settings turned on for experimental task.
With this, DMS will perform validate that the target data matches the source data when the full load is complete.